packages/api-explorer/src/lib/Oas.js

```diff
@@ -5,7 +5,8 @@ class Operation {
   constructor(oas, path, method, operation) {
     Object.assign(this, operation);
     this.oas = oas;
-    this.path = path;
+    // strips trailing slash
+    this.path = path.endsWith('/') ? path.slice(0, -1) : path;
```

Comment:
Yo! `String.prototype.endsWith` isn't supported in ie11. Can you check to see if this works in ie11? I can give you access to our browserstack account. I'm hoping that we've got @babel/env setup correctly here: https://github.com/readmeio/api-explorer/blob/7963c659cd927486abf0d15b4f7a4f8622722aca/.babelrc#L3-L7 but it's possible that we need to add an exclusion to allow ie11.

Reply:
I'd be happy to test it in IE11 or just refactor this to something more universally compatible too. What would you prefer?